luxonis_train/config/predefined_models/detection_model.py

```diff
@@ -27,10 +29,14 @@ def get_variant(variant: VariantLiteral) -> DetectionVariant:
         "light": DetectionVariant(
             backbone="EfficientRep",
             backbone_params={"variant": "n"},
+            neck_params={"variant": "n", "download_weights": True},
+            head_params={"download_weights": True},
         ),
         "heavy": DetectionVariant(
             backbone="EfficientRep",
             backbone_params={"variant": "l"},
+            neck_params={"variant": "l", "download_weights": True},
+            head_params={"download_weights": False},
```

Comment:
We have a pretrained backbone for the heavy detection model, but not a pretrained head, right?

Reply:
Not yet. We've only trained light detection model on COCO so we only have "nano" variant for the backbone, neck and head. We'll be adding more gradually, as we go.

Reply:
Ok, but why do we have `"download_weights": True` here in the heavy variant?

Reply:
Oh yeah nice catch, will fix.